src/components/sharedTransition/sharedTransition.api.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "SharedTransition",
+  "category": "interaction",
+  "description": "Component for creating shared element transitions between screens",
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/SharedTransitionScreen.js",
+  "props": [],
```

Comment:
SharedTransition is a component composed useing sub-components: area, source, target. Create an api file to each of these sub-components and name them SharedTransition.<sub-component_name> and specify each props